recipes/ua-nodeset/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class UaNodeSetConan(ConanFile):
+    name = "UA-Nodeset"
+    license = ""
+    author = 'OPCFoundation'
+    description = "This repository contains UANodeSets and other normative files which are released with a specification"
+    url = 'https://github.com/OPCFoundation/UA-Nodeset'
+    default_user = "ar"
+    default_channel = "thirdparty"
+    generators = "cmake", "virtualenv"
+    settings = 'os'
+
+
+    def source(self):
+        git = tools.Git()
```

Comment:
Please use regular tarballs instead of git.
UA-Nodeset provides releases: https://github.com/OPCFoundation/UA-Nodeset/releases
We'ld like to not need git for fetching sources.
Using tarballs also allows us to check for source modifications using hashes (sha256).

Look at other recipes on how a `conandata.yml` looks like + how to use it in the `source` method.